### PR TITLE
PR-001: Vagrant

### PR DESCRIPTION
**Pull request by apabinc**
Let's configure Vagrant VM for junkbot and release TAN VM, 'cause vagrant instance takes only 40mb of ram.

### CODE REVIEW

**Comment by dkravcovs** [comment]
Good idea, but.. Here's some thoughts:
- VM's still take disk space, the base ubuntu box takes about 2-2.5 GB. I guess it's not an issue, but we could look into something smaller.
- Don't actually need apache and all the related stuff, it just runs from console.
- No idea what to do about debugging actually. How does it work if you don't actually use a web server?

File: deploy/project.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# composer update
+cd /vagrant
+composer global require "fxp/composer-asset-plugin:1.0.0"
```
Comment (dkravcovs) [inline deploy/project.sh:5]: That was just for composer/bower integration. Not really needed since we don't have bower assets here.

**Comment by apabinc** [comment]
> VM's still take disk space, the base ubuntu box takes about 2-2.5 GB. I guess it's not an issue, but we could look into something smaller

I'm not sure about this, because it wasn't downloading another box. I think it uses the same image that also TAN project uses.

> No idea what to do about debugging actually. How does it work if you don't actually use a web server?

In PHPStorm you can debug different types of application. I will try research this.

**Comment by dkravcovs** [comment]
> I'm not sure about this, because it wasn't downloading another box. I think it uses the same image that also TAN project uses.

Yep, but that's just the base image. Check under your `%HOMEPATH%/VirtualBox VM's`

**Comment by dkravcovs** [comment]
So as agreed, let's try to migrate from basic running basic VM to 
However, this will also require us to modify the Tan box.

https://docs.vagrantup.com/v2/docker/basics.html
https://docs.docker.com/installation/windows/

**Comment by apabinc** [comment]
@darkalor No problem, we'll check it as soon as possible.

**Comment by dkravcovs** [comment]
@schtanislau Think I got a solution without messing with docker. Let's discuss later.

**Comment by apabinc** [comment]
IMHO this can be closed.

**Pull request closed without merging**